### DISCUSSION TITLE
Help with PHP errors

### DISCUSSION THREAD

**Discussion by Pawz2142** [nextcloud/all-in-one#4810] [Questions] [answered]
I finally got Nextcloud aio up and running, I'm somewhat new to working with Linux and this took a good while. I'm running OMV 7 and Nextcloud aio in a Docker Compose container. I've noticed very slow progress in downloading/uploading and decided to check the logs. I also noticed syncing on the Desktop app would stop working with a 401 error that was solved by restarting the containers. Here's some of the logs from the nextcloud container that i couldn't find information about:

NOTICE: PHP message: PHP Deprecated:  Optional parameter $userId declared before required parameter $path is implicitly treated as a required parameter in /var/www/html/custom_apps/deck/lib/Service/ConfigService.php on line 234

PHP Fatal error:  Allowed memory size of 2147483648 bytes exhausted (tried to allocate 172032 bytes) in /var/www/html/lib/private/Blurhash/Listener/GenerateBlurhashMetadata.php on line 117

PHP Fatal error:  gd-webp cannot allocate temporary buffer in /var/www/html/lib/private/legacy/OC_Image.php on line 751

Any help would be appreciated.

**Comment by szaimen** [comment, marked as answer]
Hi,

> NOTICE: PHP message: PHP Deprecated: Optional parameter $userId declared before required parameter $path is implicitly treated as a required parameter in /var/www/html/custom_apps/deck/lib/Service/ConfigService.php on line 234

This needs to be fixed in the https://github.com/nextcloud/deck/ app

> PHP Fatal error: Allowed memory size of 2147483648 bytes exhausted (tried to allocate 172032 bytes) in /var/www/html/lib/private/Blurhash/Listener/GenerateBlurhashMetadata.php on line 117

This needs to reported to and fixed in https://github.com/nextcloud/server. However you might work around it by increasing the memory limit: https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-adjust-the-php-memory-limit-for-nextcloud

> PHP Fatal error: gd-webp cannot allocate temporary buffer in /var/www/html/lib/private/legacy/OC_Image.php on line 751

This needs to reported to and fixed in https://github.com/nextcloud/server.
